x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -30,6 +30,13 @@ export function hasKibanaPrivilege(
   return fleetAuthz.packagePrivileges?.endpoint?.actions[privilege].executePackageAction ?? false;
 }
 
+export function hasEndpointExceptionsPrivilege(
```

Comment:
As discussed offline: I think the logic to determine if Lists for endpint exceptions is allowed should be moved to the current module that detemines list access for the Detection engine (or somewhere similar) - here: 

https://github.com/elastic/kibana/blob/6efef0496077f4e61d49e4e43f75941ff3d98d9e/x-pack/plugins/security_solution/public/detections/components/user_privileges/use_fetch_list_privileges.ts#L25

I'm ok if you want to leave it as is for now and refactor later.

Reply:
Any feedback on this one or are still working on it?

Reply:
@paul-tavares Still working on this. There are still some missing pieces that need to be in place to restrict APIs on Serverless and not on ESS. Once everything is working with this approach I'll reach out to you and discuss the re-factor. I do agree that this should move out of here.